postgres-types/src/geo_types_04_or_05.rs

```diff
@@ -1,6 +1,11 @@
+#[cfg(all(feature = "with-geo-types-0_4", not(feature = "with-geo-types-0_5")))]
```

Comment:
The purpose of naming the features `geo-types-04` is so that multiple versions can coexist at the same time.

Reply:
@sfackler, okay! I have extracted out a new module. 🙂